/**/**:
  ros__parameters:
    accelerometer:
      iir_filter:
        enable: true
        # butterworth low-pass, 1st order, 100 Hz sampling frequency, 10 Hz cutoff frequency
        a: [1.0, -0.5095]
        b: [0.2452, 0.2452]

      notch_filter:
        enable: false
        sample_rate: 1000 # [Hz]
        frequencies: [100, 200, 300] # [Hz]
        bandwidth: 100 # [Hz]

    gyro:
      iir_filter:
        enable: true
        # butterworth low-pass, 1st order, 200 Hz sampling frequency, 10 Hz cutoff frequency
        a: [1.0, -0.7265]
        b: [0.1367, 0.1367]

      notch_filter:
        enable: false
        sample_rate: 1000 # [Hz]
        frequencies: [100, 200, 300] # [Hz]
        bandwidth: 100 # [Hz]
